content/api/commands/intercept.md

```diff
@@ -872,6 +872,10 @@ The request object (`req`) has several properties from the HTTP request itself.
    * Request URL.
    */
   url: string
+  /**
+   * URL query string as object.
+   */
+  query: Record<string, string|number>
```

Comment:
```suggestion
  query: Record<string, string>
```

I think?

Reply:
I left my comment about why `number` is need [here](https://github.com/cypress-io/cypress/pull/16916#discussion_r655820692)